### ISSUE TITLE
Uncaught TypeError: can't convert undefined to object

### ISSUE THREAD

**Issue by vendion** [Novik/ruTorrent#2623]
### Please complete the following tasks.

- [X] Web browser cache cleared
- [X] Web browser and ruTorrent version provided

### Tell us about your environment

Web Browser: Mozilla Firefox 121.0
ruTorrent: v4.2.9

### What were you trying to do?

Load ruTorrent after upgrade from v3.x of ruTorrent. The UI just hangs at the "Loading" screen with no visible logs other than the JS error.

### What us the error message?

Uncaught TypeError: can't convert undefined to object
Error: Promised response from onMessage listener went out of scope

### Give us the stack trace.

Uncaught TypeError: can't convert undefined to object
    makeContent https://192.168.86.65:8080/js/content.js?v=429:325
    <anonymous> https://192.168.86.65:8080/js/webui.js?v=429:3222
    jQuery 9
        e
        t
        setTimeout handler*Deferred/then/l/<
        c
        add
        then
        Deferred
        then
        ready
    <anonymous> https://192.168.86.65:8080/js/webui.js?v=429:3220
content.js:325:18
    makeContent https://192.168.86.65:8080/js/content.js?v=429:325
    <anonymous> https://192.168.86.65:8080/js/webui.js?v=429:3222
    jQuery 9
    <anonymous> https://192.168.86.65:8080/js/webui.js?v=429:3220
Error: Promised response from onMessage listener went out of scope background.min.js:1:34933

**Comment by vendion** [comment]
If I roll back to v3.x it works as expected.

**Comment by stickz** [comment]
This stack trace points to `theUILang.speedGraphDurationOptions` .
Let me guess. You forgot to copy the `lang` folder when upgrading versions.

**Comment by vendion** [comment]
No, I did copy the `lang` directory, in fact, I copied everything.  The way I did the upgrade was I updated `config/config.php` to point to my `rtorrent` socket file and then ran `cp -r ruTorrent-4.2.9/* ruTorrent/`

**Comment by stickz** [comment]
Please try upgrading to [v4.2.10](https://github.com/Novik/ruTorrent/releases/tag/v4.2.10). We resolved a web browser caching issue when changing versions, that could trigger this error.

**Comment by stickz** [comment]
No response from user. We presume this to be a caching issue with version upgrades which has been resolved.
